### ISSUE TITLE
C:\Windows\System32\drivers\etc not visible in CE

### ISSUE THREAD

**Issue by VADemon** [vatterspun/cubicexplorer#6]
Needs verification.

If I go to `C:\Windows\System32\drivers\` in CE then I don't see the `etc` folder (where `hosts` file is located). If I navigate there in Windows Explorer the folder shows up as it should.

Workaround: create a symlink linking to the real `etc` - CE will display the symlink fine.

Windows 7 x64
CE: v0.96.0.1582

PS: Sorry for flooding your e-mail today :)

**Comment by vatterspun** [comment]
Thanks for that ... I can't confirm the issue on Win7x86:

![CubicExplorer_3p2tehc45h](https://user-images.githubusercontent.com/1289134/81487062-d7cae680-9227-11ea-926f-0525a464977d.png)


> PS: Sorry for flooding your e-mail today :)

No, this is fantastic.  I've been over and over the possible next-steps for this project.  I thought I'd found a developer to take over two years ago but that evaporated, and then life got in the way.  I wasn't sure this was something anyone cared about and if I just needed to figure something else out.  Your posts will hopefully build some steam towards renewing the project.

**Comment by VADemon** [comment]
![2020 05 12_22-04-34__explorer](https://user-images.githubusercontent.com/1168140/81740171-cf2f1600-949c-11ea-9358-11a2dec9e0cd.png)
_Left: Windows Explorer ✔ (correct) || Right: CubicExplorer ✘ (incorrect)_

Oh wow! I wanted to make a side-by-side screenshot and this alone uncovered the issue.
I went to `C:\Windows\System32\drivers\` in both Windows Explorer (x64) and CubicExplorer (x86) and wondered why do I see a different list of files?

Since my Windows is 64-bit, "System32" is actually a folder with 64-bit components and libraries (!), whereas SysWOW64 is for 32-bit libraries (!!!).

You can see although I navigate to System32 in CE, it's showing me contents of SysWOW64 (Windows Explorer handles this correctly). That's why I didn't see the `drivers/etc/` folder, because it's ONLY present in System32, but not in SysWOW64.

TLDR: So this is an issue with Windows' 32-bit compatibility layer and since CE is 32-bit, Windows is tricking it to go down the SysWOW64 path...

**Comment by vatterspun** [comment]
> TLDR: So this is an issue with Windows' 32-bit compatibility layer and since CE is 32-bit, Windows is tricking it to go down the SysWOW64 path...

I gotcha, makes sense -- should I close the issue?

**Comment by VADemon** [comment]
No, it's not fixed :) CE should not be tricked by Windows shims if possible.
